### PR TITLE
SG-17791: 4.6 support

### PR DESCRIPTION
**Pull request by jfboismenu**
Fixes some deprecation warnings in Mari 4.6 and updates the Mari icon to 4.6's.

### CODE REVIEW

File: hooks/tk-multi-publish2/basic/collector.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def _extract_mari_thumbnail(self):
         # render the thumbnail:
         thumb = None
         try:
-            thumb = canvas.captureImage(thumb_width, thumb_height)
+            thumb = canvas.capture(thumb_width, thumb_height)
```
Comment (jfboismenu) [inline hooks/tk-multi-publish2/basic/collector.py:200]: Also available in 3.4

**Comment by jfboismenu** [comment]
I had to rewrite the `_capture` method as 4.6v2 does not accept parameters to `canvas.capture` and since the Mari doc is pretty much non-existent it seems, I'm going for try/except instead of version checks.

**Review by eshokrgozar — APPROVED** [review]
Looks good to me. Just a small change

File: hooks/tk-multi-publish2/basic/collector.py [hunk 2]
```diff
@@ -211,5 +211,40 @@ def _extract_mari_thumbnail(self):
...
+        # differentiate between the signatures with and without arguments
+        # for capture, but the module can't read the parameters from C Python
+        # methods.
+
+        # In Mari 4.6.4+ we can capture with width and height passed in
+        try:
+            return canvas.capture(thumb_width, thumb_height)
+        except Exception:
+            pass
+        else:
```
Comment (eshokrgozar) [inline hooks/tk-multi-publish2/basic/collector.py:236]: This `else` can be removed since it will never be executed

**Pull request merged**